lib/jasmine/jasmine.ts

```diff
@@ -39,14 +39,16 @@
   });
   ['it', 'xit', 'fit'].forEach((methodName) => {
     let originalJasmineFn: Function = jasmineEnv[methodName];
-    jasmineEnv[methodName] = function(description: string, specDefinitions: Function) {
-      return originalJasmineFn.call(this, description, wrapTestInZone(specDefinitions));
+    jasmineEnv[methodName] = function(description: string, specDefinitions: Function, timeout: number) {
```

Comment:
Let's either add the `timeout` param both here and in 47 or in neither, it's weird to just add it to one.